docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+
+  <!--- Run Move Base -->
+  <include file="$(find husky_navigation)/launch/move_base.launch" />
+
+  <!-- Run Frontier Exploration -->
+  <include file="$(find husky_navigation)/launch/exploration.launch" />
+</launch>
+```
+
+#### Running the demo
+
+:::note
+In each terminal window, make sure to source the catkin workspace where you built `frontier_exploration`.
+:::
+
+1. Ensure that the Husky navigation demo package is installed:
+
+    ```bash
+    sudo apt-get install ros-noetic-husky-navigation
+    ```
+
+2. Open window #1 and start the Clearpath-configured Husky simulation environment:
+
+    ```bash
+    roslaunch husky_gazebo husky_playpen.launch
+    ```
+
+3. Open window #2 and start the Clearpath-configured [RViz](http://wiki.ros.org/rviz) visualizer:
+
+    ```bash
+    roslaunch husky_viz view_robot.launch
+    ```
+
+4. Open window #3 and start the move_base demo:
+
+    ```bash
+    roslaunch husky_navigation exploration_demo.launch
+    ```
+
+5. In the RViz visualizer, make sure the visualizers in the Navigation group are enabled.
+
+6. Use the Point tool in the top toolbar to draw a closed polygon on the map that the
+   Husky should explore. Watch the terminal window for instructions.
+
+7. As the robot moves, you should see the grey static map (map topic) grow. Occasionally,
+   the gmapping algorithm will relocalize the robot, causing a discrete jump in the map->odom transform.
+
+8. When the exploration goal is complete, you will see a feedback message in the terminal
+   window. You can now issue a new exploration goal if you wish.
+
+9. To save the generated map, you can run the map_saver utility:
+
+    ```bash
+    rosrun map_server map_saver -f <filename>
+    ```
+
+---
+
+## Testing Husky     {#testing-husky}
+
+Husky robots come preinstalled with a set of test scripts as part of the ``husky_tests`` ROS package, which can be run to verify robot functionality at the component and system levels. 
+
+If your Husky does not have the ``husky_tests`` ROS package installed already, you can manually install it by opening terminal and running:
+
+```bash
+sudo apt-get install ros-noetic-husky-tests
+```
+
+### Running ROS Tests
+
+The `ros_tests` script exposes a set of interactive tests to verify the functionality of core features.
+These tests run at the ROS-level via ROS topics, and serve as a useful robot-level diagnostic tool for
+identifying the root cause of problems, or at the very least, narrowing down on where the root cause(s) may be.
+
+To run `ros_tests` on a Husky robot, open terminal and run:
+
+```bash
+rosrun husky_tests ros_tests
+```
+
+Upon running `ros_tests`, a list of available tests will be shown in a menu. From the menu, you can choose
+individual tests to run, or simply choose the option to automatically run all the tests.
+
+The details of each test are shown below.
+
+- **E-Stop Test**
+
+    Checks that the robot's E-Stop is working properly. 
```

Comment:
I think we should change this to 'Motion-Stop'
Emergency Stop has more requirements in industry.

Reply:
there are several instances of E-stop throughout, so run a find-and-replace if you decide to change it.

Reply:
I agree we should probably change it, but I'll need Joey to update the actual test itself first. I'll create a Jira for him to update the test and user manual.